src/orders/components/OrderInvoiceList/OrderInvoiceList.tsx

```diff
@@ -80,9 +82,27 @@ const OrderInvoiceList: React.FC<OrderInvoiceListProps> = props => {
           </Typography>
         ) : (
           <ResponsiveTable>
+            <TableHead>
+              <TableRow>
+                <TableCellHeader className={classes.colNumber}>
+                  <FormattedMessage
+                    defaultMessage="Invoice no"
+                    description="invoice number"
+                  />
+                </TableCellHeader>
+                {onInvoiceSend && (
+                  <TableCellHeader className={classes.colAction}>
+                    <FormattedMessage
+                      defaultMessage="Action"
+                      description="action for invoice"
+                    />
+                  </TableCellHeader>
+                )}
+              </TableRow>
+            </TableHead>
```

Comment:
Why? These sort of codes ware removed in https://github.com/mirumee/saleor-dashboard/pull/594 It shouldn't be added.

Reply:
Idk rebase gone wrong i guess